### PR TITLE
[Internal] Update BSC tokenlist (pancakeswap)

### PR DESCRIPTION
**Pull request by optout21**

### CODE REVIEW

**Comment by bassam5033** [comment]
I went through a problem, I converted ETH BSC to a wallet that does not support this network. I want to solve the problem

**Review by vikmeup — COMMENTED** [review]
can you raise a bar a bit to only include like 3-5 new tokens?

**Comment by rewardsbunny** [comment]
How can we list our token on trust wallet swap?

**Comment by Terry69T** [comment]
gh pr checkout 14101

File: blockchains/smartchain/tokenlist.json [hunk 1]
```diff
@@ -133,15 +178,33 @@
...
+                },
                 {
                     "base": "c20000714_t0xb59490aB09A0f526Cc7305822aC65f2Ab12f9723"
                 },
+                {
+                    "base": "c20000714_t0xbcf39F0EDDa668C58371E519AF37CA705f2bFcbd"
+                },
                 {
                     "base": "c20000714_t0xBf5140A22578168FD562DCcF235E5D43A02ce9B1"
                 },
```
Comment (ACCENTCOINS) [inline blockchains/smartchain/tokenlist.json:210]: [Accent coin token list.txt](https://github.com/trustwallet/assets/files/7797761/Accent.coin.token.list.txt)

**Comment by ACCENTCOINS** [comment]
AC

File: blockchains/smartchain/tokenlist.json (hunk 1 above)
Comment (ACCENTCOINS) [inline blockchains/smartchain/tokenlist.json:210]: {
            "chainId": 56,
            "asset": "0x31aa2408a8970d7348f6d5fe4b4bc0fc38f08347",
            "type": "BEP20",
            "address": "0x31aa2408a8970d7348f6d5fe4b4bc0fc38f08347",
            "name": "AccentCoin",
            "symbol": "AC",
            "decimals": 18,
            "logoURI": "https://assets.trustwalletapp.com/blockchains/smartchain/assets/0x31aa2408a8970d7348f6d5fe4b4bc0fc38f08347/logo.png",
            "pairs": []
        },

**Comment by Wak231** [comment]
Screenshot_20220119-114631
Screenshot_20220119-114619
Screenshot_20220119-090735
Screenshot_20220119-090730
Screenshot_20220119-114638
 Bro plz help me i cant find them in my wallet

**Pull request closed without merging**